docs/guide/cloud_concepts/first-steps.md

```diff
@@ -0,0 +1,132 @@
+# Prefect Cloud: First Steps
+
+Welcome to Prefect Cloud! Now that you have an account, this guide will help you orient yourself and understand the minor adjustments required to deploy your Flows to Cloud.
+
+## Prefect Agent
+
+_Documentation Coming Soon!_
+
+## Create a Project
+
+Before deploying your first Flow, you should create a Project in Prefect Cloud.  Recall that Projects are simply an organizational tool for your Flows, and can be thought of as a directory structure.
+
+There are two simple ways of creating projects, depending on your personal preference:
```

Comment:
@cicdw What are your thoughts on also having a CLI command for this?

Reply:
Definitely not against it - seems like it could be useful; would it require a new CLI group though?

Reply:
Yeah it would. There are a small subset of quick and easy mutations that should be supported. I'll open an issue for this first one